src/frontend/src/components/SearchInput/SearchInput.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+
+const useStyles = makeStyles((theme) => ({
+  input: {
+    fontSize: '1.6rem',
+    '&:hover': {
+      border: '2px solid',
+      borderColor: theme.palette.background.default,
+    },
+    '&:focus': {
+      border: '2px solid',
+      borderColor: theme.palette.primary.main,
+    },
+    '& > *': {
+      fontSize: '1.6rem !important',
+      color: theme.palette.text.default,
+    },
+    height: '55px',
+    backgroundColor: theme.palette.background.default,
+    paddingLeft: '10px',
+    // The border around the textField inputs was a default of the type so I had to add my own
+    border: '1px solid #B3B6B7',
+    borderRadius: '7px',
+  },
+}));
+
+function AuthorSearchInput(props) {
```

Comment:
Move this function and its prop types to its own file: ` src/frontend/src/components/SearchInput/AuthorSearchInput.jsx` and import it at the top of this file.

Reply:
Done.